### DISCUSSION TITLE
Admin command filter options for trigger/event tasks

### DISCUSSION THREAD

**Discussion by Buck-Irl** [gameserverapp/Platform#881] [Suggestions]
Hi Max, 

As description says, would it be possible to add admin commands as event trigger.

I'm not sure what way filter would work. 

But I Would love an option for all admin commands,
So if any are used, Player custom role--> doesn't have --> admin 
Kick from server and send discord message--> {character.id} kicked for using admin commands.

Also an option where we could put commands in a text box like for "chat contains x" 

This box could then be set as allowed commands or disallowed commands, maybe? 
Eg. Admin command contains "gmsummon"  "forcejointribe" ---> Send discord message--> Admin {character.id} has used unauthorised commands. 
Or, Admin command contains "gcm" "ghost".  Any commands used other then those listed --> Send discord message--> Admin {character.id} has used unauthorised commands.

Maybe a variable to add {command.used} to output message? I know I may be pushing it with the variable lol.

I hope this all makes sense to you. I'm sure you could implement  it much cleaner then what I'm describing.  And since all commands log on GSA already, hopefully this isn't anything crazy I'm looking for.

Thanks for looking either way.
Buck

**Comment by mvaessen** [comment]
Nice suggestion! Thanks for the details, that helps and they do make sense to me 👍

**Reply by Buck-Irl** [reply]
Excellent,  once it makes sense

**Comment by mvaessen** [comment]
Forgot to close this one, this has been added a while back
